DirectProgramming/C++SYCL_FPGA/Tutorials/Features/kernel_args_restrict/src/kernel_args_restrict.cpp

```diff
@@ -49,42 +79,73 @@ void RunKernels(size_t size, std::vector<int> &in, std::vector<int> &nr_out,
               << std::endl;
 
     buffer in_buf(in);
-    buffer nr_out_buf(nr_out);
-    buffer r_out_buf(r_out);
 
+    // Below, we submit two SYCL kernels defined in lambda coding style.
     // submit the task that DOES NOT apply the kernel_args_restrict attribute
-    auto e_nr = q.submit([&](handler &h) {
+    buffer conservative_lambda_out_buf(conservative_lambda_out);
+    auto e_conservative_lambda = q.submit([&](handler &h) {
       accessor in_acc(in_buf, h, read_only);
-      accessor out_acc(nr_out_buf, h, write_only, no_init);
+      accessor out_acc(conservative_lambda_out_buf, h, write_only, no_init);
 
-      h.single_task<KernelArgsNoRestrict>([=]() {
+      h.single_task<IDConservative_Lambda>([=]() {
         for (size_t i = 0; i < size; i++) {
           out_acc[i] = in_acc[i];
         }
       });
     });
 
     // submit the task that DOES apply the kernel_args_restrict attribute
-    auto e_r = q.submit([&](handler &h) {
+    buffer restrict_lambda_out_buf(restrict_lambda_out);
+    auto e_restrict_lambda = q.submit([&](handler &h) {
       accessor in_acc(in_buf, h, read_only);
-      accessor out_acc(r_out_buf, h, write_only, no_init);
+      accessor out_acc(restrict_lambda_out_buf, h, write_only, no_init);
 
-      h.single_task<KernelArgsRestrict>([=]() [[intel::kernel_args_restrict]] {
+      h.single_task<IDKernelArgsRestrict_Lambda>([=
```

Comment:
```suggestion
      h.single_task<IDKernelArgsRestrict_Lambda>([=]() 
      [[intel::kernel_args_restrict]] { // NO-FORMAT: Attribute
```
clang format probably did this

Reply:
Nice catch, thx!